modules/microshift-blocking-nodeport-access.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-blocking-nodeport-access_{context}"]
+= Blocking external access to the NodePort service on a specific host interface
+
+OVN-Kubernetes does not restrict the host interface where a NodePort service can be accessed from outside a {product-title} node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
+
+.Procedure
+. Change the NODEPORT variable to the host port number assigned to your kubernetes NodePort service 
++
+[source,terminal]
+----
+$ export NODEPORT=30700
+----
+. Change the INTERFACE_IP value to the IP address from the host interface that you want to block. 
++
+[source,terminal]
+----
+$ export INTERFACE_IP=192.168.150.33
+----
+. Insert a new rule in the `nat` table PREROUTING chain to drop all packets that match the destination port and ip. 
++
+[source,terminal]
+----
+$ sudo nft -a insert rule ip nat PREROUTING tcp dport $NODEPORT ip daddr $INTERFACE_IP drop
+----
+
+. List the new rule with the following command:
++
+[source,terminal]
+----
+$ sudo nft -a list chain ip nat PREROUTING
+table ip nat {
+	chain PREROUTING { # handle 1
+		type nat hook prerouting priority dstnat; policy accept;
+		tcp dport 30700 ip daddr 192.168.150.33 drop # handle 134
+		counter packets 108 bytes 18074 jump OVN-KUBE-ETP # handle 116
+		counter packets 108 bytes 18074 jump OVN-KUBE-EXTERNALIP # handle 114
+		counter packets 108 bytes 18074 jump OVN-KUBE-NODEPORT # handle 112
+	}
+}
```

Comment:
Please double check your IDE's whitespace handling. This should be fixed throughout. I just noticed the extra line when I made a comment on line 49.

![image](https://user-images.githubusercontent.com/82119979/232599357-f8aab06f-bb1d-4095-ad27-5f83973d11de.png)

Reply:
Ah, I see it now! Fixing!